Update repository URLs for dependencies

EmmyGH opened the pull request:
Build aborts with 525 errors, but at least all dependencies are retrieved now!

emmyoh committed:
Update repository URLs for dependencies

Build aborts with 525 errors, but at least all dependencies are retrieved now!

ghost commented:
I've tried regenerating the bindings, and I'm not sure if I'm doing it right, but whatever I'm doing isn't fixing the build errors?

I've run `make gir` in the root directory, which seems to be regenerating the bindings, and then running `cargo fmt` on them. This produces no difference with the current contents of `src/auto` in `master`.

---

EDIT: Never mind, I figured it out. I had to update the `gir` and `gir-files` submodules. Adding regenerated bindings in a new commit to this PR.

emmyoh committed:
Regenerated bindings

Now build aborts after three errors

ghost commented:
It appears that the three remaining build errors are due to the fact that `glib::translate::ToGlib` no longer exists?

---

Ah, it appears it's now called `into_glib`. I've manually changed that, and now I'm getting 13 errors because `gtk::{ContainerExt, WidgetExt}` and `webkit2gtk::{SettingsExt, WebContextExt, WebViewExt}` cannot be found.

emmyoh committed:
Manually fixed `gtk` references

`ToGlib` -> `IntoGlib`
`gtk::{ContainerExt, WidgetExt}` -> `gtk::prelude::{ContainerExt, WidgetExt}`

Now, building still fails because `SettingsExt`, `WebContextExt`, and `WebViewExt` don't exist in the `webkit2gtk` crate

ghost commented:
Hopefully I'm helping in some manner? While I don't think I can progress further on my own as I'm unfamiliar with the inner workings of the GTK-rs project, I do think I've laid the groundwork for someone else to pick up where I left off

emmyoh committed:
Corrected visibility of `Ext` traits

This addresses #93 and builds upon the previous commits to this pull request.

Bear in mind that I have no idea what I'm doing—I've had to manually edit many automatically generated files throughout this PR (which I know I shouldn't have to be doing) to get this to work, but, hey, it works!

There are now eight warnings, and no errors.

I'm not sure if this can be merged just yet, for reasons such as code style and the whatnot; a review is still required

ghost commented:
I believe I'm pretty much done, as of my latest commit? I'm seeking outside review and tweaks, and hopefully this PR will get merged. Build errors have been resolved, and closes: #93.
